pennylane_lightning/lightning_gpu/lightning_gpu.toml

```diff
@@ -58,7 +58,8 @@ MultiControlledX       = {}
 # Gates which should be translated to QubitUnitary
 [operators.gates.matrix]
 
-
+BlockEncode            = {properties = [ "controllable" ]}
+'C(GlobalPhase)'       = {}
```

Comment:
The PR doesn't mention changes to the global phase, we should probably include it in description/changelog if intentional.

The syntax is weird though, I don't think we want `C(op)` to appear in the toml file.

Reply:
This is not related to this PR and all Catalyst LK tests pass without adding it. I'll remove it.

My motivation behind this update: LK and LGPU support `GlobalPhase` natively while translating `C(GlobalPhase)` to a QubitUnitry. I was looking for a way to encode this in the toml files without breaking Catalyst. The alternative that I just tested and it passed is by adding `GlobalEncode            = {properties = [ "controllable" ]}` here instead of `C(HGlobalPhase)`.

Reply:
If we can confirm that adding the controllable property in the matrix section does what we expect that's a really nice idea :)